training.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		A494E19638318FB516AC864A /* Pods_training.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 99C5C6D8C4CF860E6F78E60B /* Pods_training.framework */; };
```

Comment:
application doesn't compile

Reply:
I fixed it. There was some missing file which I don't add to git. Now it should be compiled without any errors. I checked it locally and everything should work.